submit.py

```diff
@@ -237,16 +253,14 @@ def _process_output(self, model_output):
         return carla.VehicleControl(throttle=throttle, steer=steering, brake=brake)
 
     def _visualise_outputs(self, model_input, model_output, timestamp):
-
         # Very temporary, will clean up.
         save_path = './vis_tmp'
+        os.makedirs(save_path, exist_ok=True)
```

Comment:
This comment is bad because:
- the suggested change doesn't match the comment
- we shouldn't make testing comments on PRs that don't touch tests
- even in the right context, I don't think testing this would be useful